stable/terracotta/README.md

```diff
@@ -0,0 +1,82 @@
+# Terracotta
+
+The [Terracotta 5.x OSS](http://www.terracotta.org/) offering includes the following:
+
+ *  Ehcache 3.x compatibility
+ *  Distributed In-Memory Data Management with fault-tolerance via Terracotta Server (1 stripe – active with optional mirror)
+ *  In memory off-heap storage - take advantage of all the RAM in your server
+
+
+## Quick Start
+
+```bash
+$ helm install stable/terracotta
+```
+
+## Introduction
+
+This chart bootstraps a [Terracotta server](https://github.com/Terracotta-OSS/docker) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.10+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/terracotta
+```
+
+The command deploys Terracotta on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Terracotta chart and their default values.
+
+| Parameter                                  | Description                                                                                                    | Default                                              |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
+| `image.repository`                         | Terracotta Image name                                                                                          | `terracotta/terracotta-server-oss`                   |
+| `image.tag`                                | Terracotta Image tag                                                                                           | `{VERSION}`                                          |
+| `image.pullPolicy`                         | Image pull policy                                                                                              | `Always`                                             |
+| `replicaCount`                             | Number of Terracotta members                                                                                   | 2                                                    |
+| `offheaps`                                 | Offheap resource defintions                                                                                    | offheap-1, 512, MB and offheap-2, 256, MB            |
```

Comment:
the values should be a list here too to make it more apparent

Reply:
Also please add rbac now as it's one of the requirements: https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md - shouldn't take too much effort